src/Aws/EventBridgeClient.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Common\Aws;
+
+use Aws\EventBridge\EventBridgeClient as AWSEventBridgeClient;
+
+/**
+ * Class EventBridgeClient
+ * @package Common\Aws
+ */
+class EventBridgeClient extends AWSEventBridgeClient
+{
+    public function publishEvent(string $type, array $data, array $entryParams = [])
+    {
+        $meta = [];
+        if (class_exists('DDTrace\\GlobalTracer')) {
+            $meta = [
+                'meta' => [
+                    'DD_TRACE_ID' => \DDTrace\trace_id(),
+                    'DD_SPAN_ID' => \dd_trace_peek_span_id(),
+                ]
+            ];
+        }
+
+        $event = [
+            'Entries' => [
+                array_merge([
+                    'DetailType' => $type,
+                    'Detail' => json_encode(array_merge($data, $meta)),
+                    'EventBusName' => getenv('AWS_EVENT_BUS'),
+                    'Source' => getenv('WHERE_THE_F_SHOULD_I_KNOW_THIS_FROM?!'),
```

Comment:
good question, for the node services I found there is a usage of [such env variable](https://github.com/billysbilling/billy-node-service/pull/290/commits/5d9202e4c6d7b5bcc8efbbdc3366d1ef67d68ec7#diff-69e557c9b61f142832f7a8aa5fe17c8a8f1d612f4c4efccdd07f744d06646c5dR20): `npm_package_name` - but I don't think it'll be the same for php :(

maybe mesos sets some variable?

Reply:
Right 👍🏼